.github/workflows/update-contributors.yml

```
@@ -29,12 +29,24 @@ jobs:
         run: |
           python update_contributors.py
       # Checkout and sync the update-contributors branch first
+      # Fetch all remote branches. Ignore errors if the branch doesn't exist yet.
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 32: did not find expected key

(syntax-check)